### PR TITLE
Update FAQ - changed OSS acronym to open source

### PR DESCRIPTION
**Pull request by Wasabi375**
https://discuss.kotlinlang.org/t/what-does-the-acronym-oss-mean/18556?u=wasabi375

Especially for new programmers the term OSS might not be well known. Also it's most often used as a noun and not as an adjective. Because of that it's even more confusing.

### CODE REVIEW

File: pages/docs/reference/faq.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ title: FAQ
 
 ### What is Kotlin?
 
-Kotlin is an OSS statically typed programming language that targets the JVM, Android, JavaScript and Native. 
+Kotlin is an open-source statically-typed programming language that targets the JVM, Android, JavaScript and Native.
```
Comment (p7nov) [inline pages/docs/reference/faq.md:12]:
```suggestion
Kotlin is an open-source statically typed programming language that targets the JVM, Android, JavaScript, and Native. 
```

There is no hyphen in `statically typed` (see Microsoft or Google docs)

File: pages/docs/reference/faq.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ much easier and lower-risk. There’s also an automated Java-to-Kotlin converter
 
 ### What can I use Kotlin for?
 
-Kotlin can be used for any kind of development, be it server-side, client-side web and Android. With Kotlin/Native currently 
+Kotlin can be used for any kind of development, be it server-side, client-side, web and Android. With Kotlin/Native currently
```
Comment (p7nov) [inline pages/docs/reference/faq.md:43]:
```suggestion
Kotlin can be used for any kind of development, be it server-side, client-side web and Android. With Kotlin/Native currently 
```
`client-side web` is one item of this list, so we don't need this comma.

File: pages/docs/reference/faq.md (hunk 1 above)
Comment (Wasabi375) [reply pages/docs/reference/faq.md]: Didn't know that. I used the text suggested on the kotlin forum, but I should have checked that before changing it. Not sure about the comma after JavaScript though. I'm not a native English speaker but I  don't remember every seeing a comma before an and.

Comment (p7nov) [reply pages/docs/reference/faq.md]: This comma is not mandatory but is widely used. It is called [_serial comma_](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) or Oxford comma.

**Pull request merged**